lib/files.ncl

```diff
@@ -72,7 +92,12 @@ let regenerate_files | Files -> nix.derivation.Derivation
           }
             file_descr.materialisation_method
         in
-        nix-s%"regenerate_function "%{copy_command}" "%{file_descr.file}" "%{file_descr.target}""%
+        let shell_escape = fun path =>
+          path
+          |> std.string.replace "\\" "\\\\"
```

Comment:
```suggestion
```

And if we use `'..'` as suggested below, I also think we don't need to escape `\` (just the sequence `\'` which is the only one to have a special meaning).

Reply:
Thanks for the suggestion. As the new PR uses `'…'` (single quotes) as suggested, there is no need to escape `\` (backslash).